torch/csrc/jit/script/tree_views.h

```diff
@@ -465,6 +466,30 @@ struct For : public Stmt {
   }
 };
 
+//TODO: supports only single comprehension for now
+struct ListComp : public Expr {
+  explicit ListComp(const TreeRef& tree) : Expr(tree) {
+    tree->match(TK_LIST_COMP);
+  }
+  Expr elt() const {
+    return Expr(subtree(0));
+  }
+  Expr target() const {
+    return Expr(subtree(1));
+  }
+  Expr iter() const {
+    return Expr(subtree(2));
+  }
+  // TODO: no ifs for now
```

Comment:
What does the error message look like if the user uses one of the unsupported features?